pkgs/development/python-modules/pysvn/default.nix

```diff
@@ -1,59 +1,85 @@
 { stdenv
+, lib
 , buildPythonPackage
 , fetchurl
-, pkgs
 , isPy3k
 , python
+, apr
+, aprutil
+, bash
+, e2fsprogs
+, expat
+, glibcLocales
+, neon
+, openssl
+, pycxx
+, subversion
 }:
 
 buildPythonPackage rec {
   pname = "pysvn";
-  version = "1.8.0";
-  disabled = isPy3k;
+  version = "1.9.12";
   format = "other";
 
   src = fetchurl {
     url = "http://pysvn.barrys-emacs.org/source_kits/${pname}-${version}.tar.gz";
-    sha256 = "0srjr2qgxfs69p65d9vvdib2lc142x10w8afbbdrqs7dhi46yn9r";
+    sha256 = "sRPa4wNyjDmGdF1gTOgLS0pnrdyZwkkH4/9UCdh/R9Q=";
   };
 
-  buildInputs = [ pkgs.subversion pkgs.apr pkgs.aprutil pkgs.expat pkgs.neon pkgs.openssl ]
-    ++ (if stdenv.isLinux then [pkgs.e2fsprogs] else []);
+  buildInputs = [ bash subversion apr aprutil expat neon openssl ]
+    ++ lib.optionals stdenv.isLinux [ e2fsprogs ];
 
   # There seems to be no way to pass that path to configure.
-  NIX_CFLAGS_COMPILE="-I${pkgs.aprutil.dev}/include/apr-1";
+  NIX_CFLAGS_COMPILE="-I${aprutil.dev}/include/apr-1";
+
+  prePatch = ''
```

Comment:
It'd be nice to explain this if possible.

Reply:
Not needed anymore in fact, I remove this line

Reply:
Then please squash that last commit into the commit that touches this file.

Reply:
@freezeboy so do you intend to remove this prePatch ? Even if not, perhaps since it's tests related it'd be best to use `postPatch`, per https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/patch-phase.md .

Reply:
Oh I thought your note was about the cflags, indeed I can move this sed call to postPatch